supporter-product-data/data-extracts/query-csv.sh

```diff
@@ -0,0 +1,5 @@
+ # To run this you will need to install Q http://harelba.github.io/q/
```

Comment:
Q is a tool which allows you to run sql queries against a csv file, it's been very useful in this work